api/V1/Recording.js

```diff
@@ -0,0 +1,201 @@
+var models = require('../../models');
+var util = require('./util');
+var passport = require('passport');
+var log = require('../../logging');
+var tagsUtil = require('./tagsUtil');
+var requestUtil = require('./requestUtil');
+var responseUtil = require('./responseUtil');
+var multiparty = require('multiparty');
+var config = require('../../config/config')
+var randomstring = require("randomstring");
+var AWS = require('aws-sdk');
+
+module.exports = (app, baseUrl) => {
+  var apiUrl = baseUrl + '/recordings';
+
+  /**
+   * @api {post} /api/v1/recordings Add a new recording.
+   * @apiName PostRecording
+   * @apiGroup Recordings
+   * @apiDescription This call is the new way to upload any type of recording.
```

Comment:
I don't think this should be described as "the new way". That will be confusing after a few weeks/months.

Just say something like: `This is call is used for uploads of new recordings. It currently supports raw thermal video but will eventually support all recording types.`